File: array/ts/shuffle_the_array.ts
function shuffle(nums: number[], n: number): number[] {
    const ans: number[] = []
   let  i = 0;
   let j = n;

   while(i < n) {
    ans.push(nums[i]);
    ans.push(nums[j]);
        i+=1;
        j+=1;

   }

   return ans;
   
};

File: array/python/shuffle_the_array.py
from typing import List
class Solution:
    def shuffle(self,nums: List[int],n: int) -> List[int]:
        ans = []
        i,j = 0,n
        while i < n:
            ans.append(nums[i])
            ans.append(nums[j])
            i+=1
            j+=1
        return ans